cloud-config/run-scripts/ami-preapp.sh

```diff
@@ -10,6 +10,10 @@ fi
 echo -e "\n\t$ENCD_INSTALL_TAG $(basename $0) Running"
 
 # Script Below
+# Update ulimit hard and soft openfile/nofile limits
+echo '* soft nofile 4096' | sudo -u root tee -a /etc/security/limits.conf
+echo '* hard nofile 4096' | sudo -u root tee -a /etc/security/limits.conf
```

Comment:
unrelated testing, need to remove

Reply:
I am confused, if this is unrelated testing, why is it here?

Reply:
I did a self review of my PR and found some code I forgot to remove.  I flagged it so other devs will know to ignore it.